test/e2e/util.go

```diff
@@ -576,6 +540,53 @@ func RunRC(c *client.Client, name string, ns, image string, replicas int) error
 	return nil
 }
 
+func ResizeRC(c *client.Client, ns, name string, size uint) error {
+	By(fmt.Sprintf("Resizing replication controller %s to %d", name, size))
+	resizer, err := kubectl.ResizerFor("ReplicationController", kubectl.NewResizerClient(c))
+	if err != nil {
+		return err
+	}
+	waitForReplicas := kubectl.NewRetryParams(5*time.Second, 5*time.Minute)
```

Comment:
I think we should keep such methods as simple as possible ie. when someone is writing e2e test he shouldn't care about waiting timeout. Also it's only e2e so we don't care that much about performance, so let's leave it as it is now. Function waitForRCPodsRunning is just factored out from ResizeRC to make code more readable and may be used in a different methods in util.go.